recipes/rbc/meta.yaml

```diff
@@ -41,7 +41,6 @@ about:
   home: https://github.com/xnd-project/rbc
   license: BSD-3-Clause
   license_family: BSD
-  license_file: LICENSE.txt
```

Comment:
You still need to keep this line and conda-build will find it from the source tarball

Reply:
Fixed, sorry for the confusion.